src/DataLayer/helpers/tests/filters.spec.ts

```diff
@@ -159,152 +159,158 @@ describe('mergeFilters', () => {
 });
 
 describe('handleMeasureDateFilter', () => {
-    const measureWithDateFilter: AFM.IMeasure = {
-        localIdentifier: 'm1',
-        definition: {
-            measure: {
-                item: {
-                    identifier: 'm1'
-                },
-                filters: [
-                    {
-                        absoluteDateFilter: {
-                            dataSet: {
-                                identifier: 'd2'
-                            },
-                            from: '1',
-                            to: '2'
-                        }
-                    }
-                ]
-            }
+    const dateFilterD1A: AFM.DateFilterItem = {
```

Comment:
D1A its not quite straightforward what it is, please could you pick some more explicit name ? :)